crates/cli/tests/cli_run.rs

```diff
@@ -397,14 +408,19 @@ mod cli_run {
         )
     }
 
+    #[cfg(windows)]
+    const LINE_ENDING: &str = "\r\n";
+    #[cfg(not(windows))]
+    const LINE_ENDING: &str = "\n";
+
     #[test]
     // uses C platform
     fn platform_switching_main() {
         test_roc_app_slim(
             "examples/platform-switching",
             "main.roc",
             "rocLovesPlatforms",
-            "Which platform am I running on now?\n",
+            &("Which platform am I running on now?".to_string() + LINE_ENDING),
```

Comment:
really strange how this is needed. we just write the bytes of the rocstr (which ends in "now?\n")  to stdout, but somehow that `\r` makes it in.

Reply:
Yeah I'd think this would *break* it! How strange haha